gnmi_server/server.go

```diff
@@ -177,13 +177,13 @@ func (srv *Server) Serve() error {
 	return srv.s.Serve(srv.lis)
 }
 
-func (srv *Server) Stop() {
+func (srv *Server) GracefulStop() {
```

Comment:
Do we really need to rename this API?

Reply:
Won't use Graceful stop in this PR, but yes, we don't have to change the name of the wrapper API